src/systems/ec/implementations/vanilla/packages/ec-component-gallery/ec-component-gallery.scss

```diff
@@ -325,8 +330,7 @@
       justify-content: space-between;
     }
 
-    .ecl-gallery__slider-image-container, // DEPRECATED
-    .ecl-gallery__slider-media-container {
+    .ecl-gallery__slider-image-container { // DEPRECATED
```

Comment:
@emeryro needed to remove this property for `.ecl-gallery__slider-media-container` to correctly display the iframe, why do we have this declared?

Reply:
I don't really remember what this css is about...
I didn't found any issue whith this modification, so it should be fine. I just hope that it was there because of some weird display issue on Safari or Internet Explorer

Reply:
Tested on IE11 and Safari and I can confirm everything work as expected.